### PR TITLE
fix highlight of TJoe.io links

### PR DESCRIPTION
**Pull request by lucasew**
Not a common reality on Windows shells but many
Linux shells have a feature that you can click,
or Ctrl+Click on links to open them but right
now the links to TJoe.io are incomplete and not
recognized as links by the shell.

### Related Issue/Addition to code

- Fixes #
- Addition to code.

#### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Proposed Changes
- Add https prefixes to TJoe.io URLs

### Why is this change needed?
 
One can now click, or Ctrl+click on links instead of copypasting them on supported shells such as Terminator or Kitty

### Additional Info
I tested it on Kitty and now clicking on links works!

### Checklist:

- [x] My code follows the style guidelines of this project and I have read [CONTRIBUTING.md](/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

---

### Screenshots

Original | Updated
:----------------------:|:-----------:
** image  | ** image **

### CODE REVIEW

File: assets/cancelButton.png
Comment (Firecul) [inline assets/cancelButton.png:1]: Why is this being deleted?

File: assets/executeButton.png
Comment (Firecul) [inline assets/executeButton.png:1]: Why is this being deleted?

File: assets/icon.png
Comment (Firecul) [inline assets/icon.png:1]: Why is this being deleted?

File: assets/inputSubmitButton.png
Comment (Firecul) [inline assets/inputSubmitButton.png:1]: Why is this being deleted?

File: assets/inputTextBox.png
Comment (Firecul) [inline assets/inputTextBox.png:1]: Why is this being deleted?

File: assets/outputTextBox.png
Comment (Firecul) [inline assets/outputTextBox.png:1]: Why is this being deleted?

**Review by Firecul — CHANGES_REQUESTED** [review]
I have no problem with the intent of this MR and I think it's a good idea. However, I think there is some thing going wrong in your set up as it is wiping the images to nothing also.

**Comment by lucasew** [comment]
Nice catch. I didn't realize that. I am going to revert it later. Basically I just ran a search/replace using a script in the whole repo.

**Comment by lucasew** [comment]
Solved. No idea about the reason of the issue but now it's solved!

**Review by Firecul — APPROVED** [review]
Looks better now 👍

**Comment by ThioJoe** [comment]
That's the point of the non shortened link right below it though 🤔

I specifically wanted the link to be one you can remember and type into a browser if your terminal doesn't support clicking links. Adding https to the shortened link would suggest otherwise to the user.

So I consider this working as intended.

**Pull request closed without merging**